mediaindexer: Fix naming convention and permissions for ACG

Herrie82 opened the pull request:
Solves:

Jun 11 11:08:16 qemux86-64 mojodb-luna[586]: [] [pmlog] <default-lib> LS_REQUIRES_SECURITY {"SERVICE":"org.webosports.mediaindexer","CATEGORY":"/","METHOD":"find"} Service security groups don't allow method call.

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>

Tofee commented:
You're changing the service name, but I still see it in the code: https://github.com/webOS-ports/mediaindexer/blob/master/src/MediaScannerServiceApp.cc#L27
Is that intended ?

Herrie82 commented:
> You're changing the service name, but I still see it in the code: https://github.com/webOS-ports/mediaindexer/blob/master/src/MediaScannerServiceApp.cc#L27
> Is that intended ?

See last file in PR, changed it there... That was the initial problem as well and then I figured and fixed it ;)
